### PR TITLE
SSO-and-AzureAD

### PR DESCRIPTION
**Pull request by Jesse-Ben**

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/885#" title="Author self-approved">Jesse-Ben</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
Hi @Jesse-Ben. Thanks for your PR.

I'm waiting for a [spectrocloud](https://github.com/orgs/spectrocloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/spectrocloud/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: content/docs/08-user-management/4-saml-sso.md [hunk 1]
```diff
@@ -39,35 +39,40 @@ The two types of SSO authentication methods Palette supports are: <p></p><br />
...
+   - Login URL
+   - FirstName
+   - LastName
+   - Email
+   - SpectroTeam
+   - Service Provider Metadata
+
+<p></p>
+
+5. Copy the **Identity Provider Metadata** from the IdP into the Palette SAML panel for each metadata. {There is no Metadata? Do we need this step?}
```
Comment (Jesse-Ben) [inline content/docs/08-user-management/4-saml-sso.md]: Yes, and good question. Tis is needed if you use SAML, and left blank if you use OIDC.

File: content/docs/08-user-management/4-saml-sso.md [hunk 2]
```diff
@@ -6,99 +6,85 @@ icon: ""
...
-   - Login URL
-   - FirstName
-   - LastName
-   - Email
-   - SpectroTeam
-   - Service Provider Metadata
+    The following parameters will enable Spectro Cloud Palette as a **Service Provider** (SP) in your IdP. Your IdP will require some or all the information listed below to enable SSO with Palette. <p></p><br />
 
-<p></p>
+      - **Single Logout URL** - This is the logout URL. Include the URL into the IdP for the logout to clear the provider-side session. <p></p><br />
```
Comment (Jesse-Ben) [inline content/docs/08-user-management/4-saml-sso.md:51]: Change to "- **Single Logout URL** - The IdP will need the logout URL for the SSO configuration of SAML. <p></p><br />

**Review by Jesse-Ben — COMMENTED** [review]
I made a few suggestions. Please review.

**Review by Siboney1 — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/885#" title="Author self-approved">Jesse-Ben</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/885#pullrequestreview-1126148843" title="Approved">Siboney1</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**